enterprise/users/src/main/java/io/crate/metadata/sys/SysPrivilegesTableInfo.java

```diff
@@ -87,7 +88,7 @@ public RowGranularity rowGranularity() {
     }
 
     @Override
-    public Routing getRouting(WhereClause whereClause, @Nullable String preference) {
+    public Routing getRouting(WhereClause whereClause, @Nullable String preference, @Nullable SessionContext context) {
```

Comment:
I don't think it's a good idea to bloat up the dependencies for the sake of "we might need this some time in the future". YAGNI